<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDaoYouGmUi</class>
 <widget class="QWidget" name="CDaoYouGmUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineItemName">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>17</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>回车马上将GM命令拷贝到剪切板</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>GM命令</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textGm">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>231</width>
     <height>131</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineNum">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>17</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1000000</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCopy">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>物品名称</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineItemName</tabstop>
  <tabstop>lineNum</tabstop>
  <tabstop>textGm</tabstop>
  <tabstop>btnCopy</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
